src/windows/views/emojis_listview.py

```diff
@@ -109,48 +106,49 @@ def add_file(self, filepath):
             # Log exception
             log.warning("Failed to import file: {}".format(str(ex)))
 
+    @pyqtSlot(int)
+    def group_changed(self, index=-1):
+        log.info("Group set to: {}".format(index))
+        item = get_app().window.emojiFilterGroup.itemData(index)
+        log.info("Filtering on {}".format(item))
+        self.group_model.setFilterFixedString(item)
+        self.group_model.setFilterKeyColumn(1)
+
+        self.refresh_view()
```

Comment:
And I also left my debug logging in, dammit. I'll fix that too.

This is the code that implements the group-selection, which like I said is now separate from text filtering. It makes it a lot simpler _and_ a lot more like what you'd expect, IMHO.

Because `group_changed` is connected to `currentIndexChanged`, it gets an index for an item in the dropdown box — so I just pull the string to match from the data element (UserRole) of that entry and filter the model with an exact string match (`setFilterFixedString`) on the selected item. (It has to be the data role, since getting the `.text()` would return the translated, reformatted display text which isn't what we want.)

Reply:
Fixed, so bumping this comment in the hopes it won't keep getting hidden as "Outdated" since it's not.